src/go/guestagent/pkg/containerd/events.go

```diff
@@ -78,6 +79,8 @@ func NewEventMonitor(
 // MonitorPorts subscribes to event API
 // for container Create/Update/Delete events.
 func (e *EventMonitor) MonitorPorts(ctx context.Context) {
+	go e.initializeRunningContainers(ctx)
```

Comment:
Shouldn't we do this _after_ we call `Subscribe`? Didn't we have a bug recently that's also due to enumerating existing things before starting to listen for changes?

Reply:
Yeah, TBH it wouldn't make much difference. The previous thing wasn't an actual bug, we just speculated a "potential timing rance", but I will move anyway.